CMakeLists.txt

```diff
@@ -692,6 +692,11 @@ if(HPX_WITH_PARCEL_COALESCING)
   hpx_add_config_define(HPX_HAVE_PARCEL_COALESCING)
 endif()
 
+# Developer tools
+hpx_option(HPX_WITH_VIM_YCM STRING
+  "Generate HPX completion file for VIM YouCompleteMe plugin and save in specified location"
+  "" ADVANCED)
```

Comment:
Are saying that the user should write cmake -DHPX_WITH_VIM_YCM=<dest-dir>? In this case this has to be reflected in the doc string of that option. Iwas assuming the option would simply enable the generation of the YCM file.